src/driver/ydb/YdbQueryRunner.ts

```diff
@@ -12,52 +12,82 @@ import { View } from "../../schema-builder/view/View"
 import { IsolationLevel } from "../types/IsolationLevel"
 import { YdbDriver } from "./YdbDriver"
 import { ReplicationMode } from "../types/ReplicationMode"
+import { QueryRunnerAlreadyReleasedError } from "../../error"
+import { Broadcaster } from "../../subscriber/Broadcaster"
 
 export class YdbQueryRunner extends BaseQueryRunner implements QueryRunner {
+    /**
+     * Database driver used by connection.
+     */
+    driver: YdbDriver
+
     constructor(ydbDriver: YdbDriver, replicationMode: ReplicationMode) {
         super()
+        this.driver = ydbDriver
+        this.connection = ydbDriver.connection
+        this.broadcaster = new Broadcaster(this)
     }
 
-    query(
+    async query(
         query: string,
-        parameters?: any[] | undefined,
+        parameters?: any | undefined,
         useStructuredResult?: boolean | undefined,
     ): Promise<any> {
-        throw new Error("Method not implemented.")
+        if (this.isReleased) throw new QueryRunnerAlreadyReleasedError()
+
+        await this.driver.driver?.tableClient.withSession(async (session) => {
+            const preparedQuery = await session.prepareQuery(query)
```

Comment:
I think we don't need this here, but we need customizable query, which can prepare query and do multiple things within one session